src/main/java/wolox/training/models/Book.java

```diff
@@ -5,117 +5,59 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
 
 @Entity
 public class Book {
     @Id
+    @Getter
```

Comment:
Instead of adding the @Getter and @Setter annotation in each field, you can add behind @Entity(class level) and lombok will implement for all the fields.